rise-storage.html

```diff
@@ -643,46 +643,14 @@
     /**
      * Processes a Storage file.
      */
-    _handleStorageFile: function(resp) {
-      var file = {},
-        etag = null;
-
-      // File in the root of the bucket.
-      if (resp.selfLink !== undefined) {
-        etag = resp.etag;
-      }
-      // File in a folder.
-      else if ((resp.files !== undefined) && (resp.files.length > 0)) {
-        etag = resp.files[0].etag;
-      }
+    _handleStorageFile: function() {
+      var file = {};
 
       file.url = this._fileUrl;
-
-      if (this._isLoading) {
-        this._files.push({
-          "name": this._getFileNameFromUrl(file.url),
-          "etag": etag
-        });
-
-        file.added = true;
-
-        this._isLoading = false;
-      }
-      else {
-        // File hasn't changed.
-        if (this._files[0].etag === etag) {
-          file.changed = false;
-        }
-        else {
-          this._files[0].etag = etag;
-          file.changed = true;
-        }
-      }
-
+      file.changed = true;
```

Comment:
@stulees what may happen is that widget get failure to retrieve the file from storage. It will ask the component for the metadata. If there is no issue with the metadata it will call this [line of code](https://github.com/Rise-Vision/rise-storage/blob/6ec5fb0412de941a6654e137db201a48ec44b54b/rise-storage.html#L570). In this case the widget will be expecting a file.changed as it was a refresh. 

That is what I see happening, what do you think? cheers